From khali@linux-fr.org Sat Nov 26 19:24:33 2005
Date: Sat, 26 Nov 2005 20:40:34 +0100
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Cc: Laurent Riffard <laurent.riffard@free.fr>, Benjamin Herrenschmidt <benh@kernel.crashing.org>
Subject: [PATCH 13/25] i2c: Drop i2c_driver.{owner,name}, 4 of 11
Message-Id: <20051126204034.4c195439.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-drop-driver-owner-and-name-04-macintosh.patch

From: Laurent Riffard <laurent.riffard@free.fr>

We should use the i2c_driver.driver's .name and .owner fields
instead of the i2c_driver's ones.

This patch updates the drivers for macintosh.

Signed-off-by: Laurent Riffard <laurent.riffard@free.fr>
Signed-off-by: Jean Delvare <khali@linux-fr.org>
Acked-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/macintosh/therm_adt746x.c        |    6 ++++--
 drivers/macintosh/therm_pm72.c           |    6 ++++--
 drivers/macintosh/therm_windtunnel.c     |    6 ++++--
 drivers/macintosh/windfarm_lm75_sensor.c |    6 ++++--
 4 files changed, 16 insertions(+), 8 deletions(-)

--- gregkh-2.6.orig/drivers/macintosh/therm_adt746x.c
+++ gregkh-2.6/drivers/macintosh/therm_adt746x.c
@@ -171,8 +171,10 @@ detach_thermostat(struct i2c_adapter *ad
 }
 
 static struct i2c_driver thermostat_driver = {  
-	.owner		= THIS_MODULE,
-	.name		= "therm_adt746x",
+	.driver = {
+		.owner	= THIS_MODULE,
+		.name	= "therm_adt746x",
+	},
 	.attach_adapter	= attach_thermostat,
 	.detach_adapter	= detach_thermostat,
 };
--- gregkh-2.6.orig/drivers/macintosh/therm_pm72.c
+++ gregkh-2.6/drivers/macintosh/therm_pm72.c
@@ -283,8 +283,10 @@ static int therm_pm72_detach(struct i2c_
 
 static struct i2c_driver therm_pm72_driver =
 {
-	.owner		= THIS_MODULE,
-	.name		= "therm_pm72",
+	.driver = {
+		.owner	= THIS_MODULE,
+		.name	= "therm_pm72",
+	},
 	.attach_adapter	= therm_pm72_attach,
 	.detach_adapter	= therm_pm72_detach,
 };
--- gregkh-2.6.orig/drivers/macintosh/therm_windtunnel.c
+++ gregkh-2.6/drivers/macintosh/therm_windtunnel.c
@@ -354,8 +354,10 @@ do_detach( struct i2c_client *client )
 }
 
 static struct i2c_driver g4fan_driver = {  
-	.owner		= THIS_MODULE,
-	.name		= "therm_windtunnel",
+	.driver = {
+		.owner	= THIS_MODULE,
+		.name	= "therm_windtunnel",
+	},
 	.id		= I2C_DRIVERID_G4FAN,
 	.attach_adapter = do_attach,
 	.detach_client	= do_detach,
--- gregkh-2.6.orig/drivers/macintosh/windfarm_lm75_sensor.c
+++ gregkh-2.6/drivers/macintosh/windfarm_lm75_sensor.c
@@ -47,8 +47,10 @@ static int wf_lm75_attach(struct i2c_ada
 static int wf_lm75_detach(struct i2c_client *client);
 
 static struct i2c_driver wf_lm75_driver = {
-	.owner		= THIS_MODULE,
-	.name		= "wf_lm75",
+	.driver = {
+		.owner	= THIS_MODULE,
+		.name	= "wf_lm75",
+	},
 	.attach_adapter	= wf_lm75_attach,
 	.detach_client	= wf_lm75_detach,
 };
